awx/api/urls/urls.py

```diff
@@ -136,6 +140,8 @@
     re_path(r'^activity_stream/', include(activity_stream_urls)),
     re_path(r'^workflow_approval_templates/', include(workflow_approval_template_urls)),
     re_path(r'^workflow_approvals/', include(workflow_approval_urls)),
+    re_path(r'^bulk/job_launch/$', BulkJobLaunchView.as_view(), name='bulk_job_launch'),
```

Comment:
did I say this on the other PR?  Anyway, I think you should break out the bulk urls into their own urls module.